pop-window.js

```diff
@@ -0,0 +1,160 @@
+    },
+    languages: {
+      featured: ['html', 'css', 'javascript'],
+      detailed: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstrap'],
+    },
+    liveurl: 'https://therealadetunji.github.io/Michael_portfolio/',
+    sourcecode: ' ',
+  },
+
+  {
+    name: 'Multi-Post Stories',
+    workdescription: {
+      company: 'FACEBOOK',
+      role: 'FullStack Dev',
+      year: '2015',
+    },
+    description: {
+      featured:
+        "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    },
+    languages: {
+      featured: ['html', 'css', 'javascript'],
+      detailed: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstrap'],
+    },
+    liveurl: 'https://therealadetunji.github.io/Michael_portfolio/',
+    sourcecode: ' ',
+  },
+
+  {
+    name: 'Facebook 360',
+    workdescription: {
+      company: 'Facebook',
+      role: 'Fullstack Dev',
+      year: '2015',
+    },
+    description: {
+      featured:
+        "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    },
+    languages: {
+      featured: ['html', 'css', 'javascript'],
+      detailed: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstrap'],
+    },
+    liveurl: 'https://therealadetunji.github.io/Michael_portfolio/',
+    sourcecode: 'not sure it',
+  },
+
+  {
+    name: 'Uber Navigation',
+    workdescription: {
+      company: 'Uber',
+      role: 'Uber Developer',
+      year: '2018',
+    },
+    description: {
+      featured:
+        "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    },
+    languages: {
+      featured: ['html', 'css', 'javascript'],
+      detailed: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstrap'],
+    },
+    liveurl: 'https://therealadetunji.github.io/Michael_portfolio/',
+    sourcecode: '',
+  },
+];
+
+console.log(data);
+const projectsection = document.querySelector('.work'); // to get the grid and working on that work section
+data.forEach((element) => {
+  // forEach or map
+  //   console.log(element);
+  const allCard = `
```

Comment:
- [ ] [OPTIONAL] the projects are not generating the proper data for each project, kindly fix this so every project has a popup with the right title, image and description